### PR TITLE
Admin/update precommit

### PR DESCRIPTION
**Pull request by benjijamorris**
## What does this PR do?
- upgrade precommit versions
- address issues found with new precommit versions

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **test your PR locally** with `pytest` command?
- [x] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: cyto_dl/models/vae/image_canon_vae.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def __init__(
                 )
             )
 
-        if isinstance(prior, (str, type(None))):
+        if prior is None or isinstance(prior, None):
```
Comment (ritvikvasan) [inline cyto_dl/models/vae/image_canon_vae.py:127]: this feels like its not checking for prior being a str?

**Review by ritvikvasan — APPROVED** [review]
noticed one weird thing, but otherwise looks good

File: cyto_dl/models/vae/image_canon_vae.py (hunk 1 above)
Comment (benjijamorris) [reply cyto_dl/models/vae/image_canon_vae.py:127]: yikes good catch

Comment (benjijamorris) [reply cyto_dl/models/vae/image_canon_vae.py]: fixed!

**Pull request merged**